### DISCUSSION TITLE
Fond something else than TAB to accept suggestion !

### DISCUSSION THREAD

**Discussion by Entretoize** [community/community#129153] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello, very often I need to press tab to indent something and copilot is showing suggestion then I need to write before and indent later to prevent suggestion to be accepted.
Find something else please...

**Comment by LLUMOAI** [comment]
Hi @Entretoize try adjusting your editor settings or using keyboard shortcuts to reduce Copilot's suggestions while indenting or consider disabling suggestions temporarily for a smoother workflow.

**Comment by ayim** [comment]
@Entretoize which IDE are you in?

**Comment by sachetsubedi** [comment]
If you are using vs code, You can assign any other key to accept suggestions by following these steps:
1. Click on the copilot menu on the bottom right of your screen
2. Select edit keyboards shortcuts option
3. Add a key that you want to use as the key to accept suggestions in the GitHub Copilot Apply Suggestions with Copilot

**Comment by Entretoize** [comment]
I mainly use vscode and I know about shortcut customization but as I suppose most user should have that issue (or I'm the only one indending ??) and it should be default to use another key because it's really annoying... but thank for your help

**Comment by Entretoize** [comment]
That said, attributing another key to accept suggestion doesn't work, nothing append, I tried several keys...
